README.md

```diff
@@ -84,7 +84,7 @@ The version and download url are optional. By default, the latest version of the
 
 The following custom version specifiers can also be used:
 
-* `latest` - downloads the latest version from a version specific update center if one exists for the version in the Jenkins war file. If no version specific update center exists, will use the main update center [https://updates.jenkins.io](https://updates.jenkins.io)
+* `latest` - downloads the latest version from a version specific update center if one exists for the version in the Jenkins war file. Unless some specific version is explicitly requested. If no version specific update center exists, will use the main update center [https://updates.jenkins.io](https://updates.jenkins.io)
```

Comment:
this is the wrong section for what you're trying to describe, it's the version 'latest'

```suggestion
* `latest` - downloads the latest version from a version specific update center if one exists for the version in the Jenkins war file. If no version specific update center exists, will use the main update center [https://updates.jenkins.io](https://updates.jenkins.io)
```

Reply:
Fixed